.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -2,4 +2,4 @@
 _Remove this and add a detailed description of the changes that were made._
 
 ## Test Plan
-_Remove this and add a test plan that the reviewer can use to test your changes._ 
+_Remove this and add a test plan that the reviewer can use to test your changes._
```

Comment:
Hey @vzajkov Can you also revert this change? You should be able to run this command to revert this file to what is in master:
`git checkout origin/master .github/PULL_REQUEST_TEMPLATE.md`

Actually, it might be slightly different since this is a forked repo and not a branch off of the HAH repo. Let me know if you run into any issues!